src/VernaForm.tsx

```diff
@@ -16,7 +16,6 @@ function VernaForm({ onSubmit }: VernaFormProperties) {
       uiSchema={uiSchema}
       formData={selectedFormData}
       onSubmit={handleSubmit(onSubmit)}
-      readonly={readOnly}
       widgets={widgets}
       FieldTemplate={RenderFieldTemplate}
```

Comment:
To complete my comment within RenderFieldTemplate 

```suggestion
      {...(isEditor && { FieldTemplate: RenderFieldTemplate })}
```

Reply:
Very strange, when i do this it doesn't rerender, and so if it was in editor mode, it continue to use RenderFieldTemplate, and if it's loaded with isEditor as false it will never update itself and never use RenderFieldTemplate.
![image](https://user-images.githubusercontent.com/26571296/158630225-8d9c0d76-91d8-4c70-9c93-a36bd7c75021.png)

Reply:
Any idea of why it wouldn't?